validator/src/bin/solana-test-validator.rs

```diff
@@ -348,7 +349,9 @@ fn main() {
         exit(1);
     });
 
-    let features_to_deactivate = pubkeys_of(&matches, "deactivate_feature").unwrap_or_default();
+    let mut features_to_deactivate = pubkeys_of(&matches, "deactivate_feature").unwrap_or_default();
+    // Remove this when client support is ready for the enable_partitioned_epoch_reward feature
+    features_to_deactivate.push(feature_set::enable_partitioned_epoch_reward::id());
```

Comment:
do we have a way to override if explicitly requested to activate?

Reply:
Only by modifying the code and building from source. Given that I think we hope to have actual first-class client support for the feature implemented soon, I thought that was sufficient. Wdyt?

Reply:
oh for some reason i was thinking we had something along the lines of `--deactivate-all-features --activate-feature <FEATURE_ADDRESS...>`.  looks like that never got implemented though. carry on!

Reply:
Ah, that rings a bell for me as well, but I don't recall what happened.